agent/config/parse_windows.go

```diff
@@ -113,3 +123,61 @@ var isWindows2016 = func() (bool, error) {
 
 	return false, nil
 }
+
+func getInstallationType(installationType string) (string, error) {
+	if strings.Compare(installationType, InstallationTypeFull) == 0 {
+		return "FULL", nil
+	} else if strings.Compare(installationType, InstallationTypeCore) == 0 {
+		return "CORE", nil
+	} else {
+		err := seelog.Errorf("Invalid Installation type")
+		return "", err
+	}
+}
+
+func GetOSFamilyType() string {
```

Comment:
It seems if any step of this GetOSFamilyType function fails it seems we will just return `unknown`, should we maybe return `windows` (ie, `config.OSType`) to keep the behavior the same as before?